packages/cli-lib/lang/en.lyaml

```diff
@@ -1002,6 +1001,10 @@ en:
             nameRequired: "A project name is required"
             locationRequired: "A project location is required"
             invalidTemplate: "[--template] Could not find template {{ template }}. Please choose an available template."
+        downloadProjectPrompt:
+          selectProject: "Select a project to download:"
+          errors:
+            projectNotFound: "Your project {{ projectName }} could not be found in {{ accountId }}"
```

Comment:
I like this pattern of validating the users input. Maybe we should add something to the end of this along the lines of "...not found in {{ accountId }}. Please select a valid project:"?

Reply:
Nice, I like that pattern!